### PR TITLE
DM-29571: Fixed the package versions so the aarch64 conda package bui…

### PR DESCRIPTION
**Pull request by wvreeven**
…lder can be built.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (tribeiro) [inline .gitignore:1]: Is this really necessary?

**Review by tribeiro — APPROVED** [review]
Before merging, can you undo the changes in `.gitignore`?

File: .gitignore (hunk 1 above)
Comment (wvreeven) [reply .gitignore:1]: I suppose not but it is helpful to me. In any case, I will revert that change.

Comment (tribeiro) [reply .gitignore]: you can always use global git settings, for stuff that useful "for you"

Comment (tribeiro) [reply .gitignore]: e.g.; `~/.gitignore_global`

Comment (wvreeven) [reply .gitignore]: Good point!

**Pull request merged**